docsrc/user_guide/saving_models.rst

```diff
@@ -42,7 +42,7 @@ b) ExportedProgram
     inputs = torch.randn((1, 3, 224, 224)).cuda()
     trt_gm = torch_tensorrt.compile(model, ir="dynamo", inputs) # Output is a torch.fx.GraphModule
     # Transform and create an exported program
-    trt_exp_program = torch_tensorrt.dynamo.transform(trt_gm, inputs, call_spec)
+    trt_exp_program = torch_tensorrt.dynamo.serialize(trt_gm, inputs, call_spec, ir="exported_program")
```

Comment:
For this modification, should there be a note in the docs that indicates this IR will fail/is not supported yet in the `torch_tensorrt.compile` path as a target IR, or would that be more confusing?